front/app/api/authentication_requirements/getAuthenticationRequirements.ts

```diff
@@ -0,0 +1,45 @@
+import fetcher from 'utils/cl-react-query/fetcher';
+import { queryClient } from 'utils/cl-react-query/queryClient';
+import {
+  AuthenticationContext,
+  AuthenticationRequirementsResponse,
+} from './types';
+import requirementsKeys from './keys';
+
+const getAuthenticationRequirements = (
+  authenticationContext: AuthenticationContext
+) => {
+  return queryClient.fetchQuery(
+    requirementsKeys.item(authenticationContext),
+    () => fetchAuthenticationRequirements(authenticationContext)
+  );
+};
+
+const fetchAuthenticationRequirements = (
+  authenticationContext: AuthenticationContext
+) => {
+  const { type, action } = authenticationContext;
+
+  if (type === 'initiative') {
+    return fetcher<AuthenticationRequirementsResponse>({
+      path: `/permissions/${action}/requirements`,
+      action: 'get',
```

Comment:
Just wanted to double check that this is the right endpoint - if we don't pass in any context then (just /permissions/action/requirements) this gets the requirements for initiatives then?

I feel like it would be clearer if we passed in "initiatives" somewhere in the URL. I wonder if in the future we might have other "global" participation methods too.. so could be nice to specify it now :thinking: